gpu.Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG BASE_TAG=staging
+ARG BASE_TAG=dlvm-pretest
```

Comment:
FYI, no need to update this base tag.

Jenkins sets the proper value: https://github.com/Kaggle/docker-python/blob/master/Jenkinsfile#L68

And if you are building the image locally, you can use the `--base-image-tag` (`-b`) flag when running the `./build` script:
https://github.com/Kaggle/docker-python/blob/master/build#L12

Reply:
Ah got it, I was doing this just for my local build - but don't need it if I can just pass that in!